README.adoc

```diff
@@ -36,60 +36,34 @@ Existing tools such as `oc` are more operations-focused and require a deep-under
 [[installing-odo]]
 == Installing `odo`
 
-*Prerequisites:*
+=== Prerequisites:
 
-* OpenShift 3.10.0 or above is required.
+OpenShift 3.10.0 or above is required.
 
-*Procedure:*
+=== Procedure:
 
-=== OS-independent automated installation
+==== OS-independent automated installation for Mac and *nix systems
 
 Use this link:./scripts/install.sh[bash script] to quickly install `odo`. It will automatically detect your operating system and install `odo` accordingly.
 
 ----
 $ curl -L https://github.com/openshift/odo/raw/master/scripts/install.sh | bash
 ----
 
-=== macOS
-
-*Binary installation:*
-
-----
-# curl -L https://github.com/openshift/odo/releases/download/v1.0.0-beta1/odo-darwin-amd64 -o /usr/local/bin/odo && chmod +x /usr/local/bin/odo
-----
-
-*Tarball installation:*
-
-----
-# sh -c 'curl -L https://github.com/openshift/odo/releases/download/v1.0.0-beta1/odo-darwin-amd64.gz | gzip -d > /usr/local/bin/odo; chmod +x /usr/local/bin/odo'
-----
-
-=== Linux
-
-*Binary installation:*
-
-----
-# curl -L https://github.com/openshift/odo/releases/download/v1.0.0-beta1/odo-linux-amd64 -o /usr/local/bin/odo && chmod +x /usr/local/bin/odo
-----
-
-*Tarball installation:*
-
-----
-# sh -c 'curl -L https://github.com/openshift/odo/releases/download/v1.0.0-beta1/odo-linux-amd64.gz | gzip -d > /usr/local/bin/odo; chmod +x /usr/local/bin/odo'
-----
+For more installation options please refer the link:docs/installation.adoc[installation page].
```

Comment:
This sentence should be deleted from here. 
The structure should be:
== Installing ODO
=== OS-independent...
=== Windows
In the concluding statement for the installation section we should add a sentence saying:
For detailed information and more installation options see the link:docs/installation.adoc[installation guide].

Reply:
If I'm a Linux user, I'd probably not bother to read what's mentioned in the Windows section. Doing what you are suggesting would put the detailed installation steps for *all* OS at the end of Windows section. I'd suggest we put it in both the "Mac & *nix" as well as "Windows" section.

Reply:
Sounds good